### DISCUSSION TITLE
Unable to get github education account although it shows that i'm accepted for 4 days

### DISCUSSION THREAD

**Discussion by YongJun05** [community/community#130678] [GitHub Education]
### Hi everyone,

please give me a feedback, thanks!

**Comment by davevad93** [comment]
Hi @YongJun05 ,

Student benefits usually take a few days to become available, but may take longer depending on various factors, such as the volume of requests. When available, the status will change to:

`Congrats! Your request for @YongJun05  was approved and your benefits are now available`
